[2023-04-09T03:27:29.101+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.store_user manual__2023-04-09T03:27:22.565474+00:00 [queued]>
[2023-04-09T03:27:29.109+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.store_user manual__2023-04-09T03:27:22.565474+00:00 [queued]>
[2023-04-09T03:27:29.110+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T03:27:29.111+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-04-09T03:27:29.112+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T03:27:29.121+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2023-04-09 03:27:22.565474+00:00
[2023-04-09T03:27:29.127+0000] {standard_task_runner.py:55} INFO - Started process 10703 to run task
[2023-04-09T03:27:29.131+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'store_user', 'manual__2023-04-09T03:27:22.565474+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmpu9ts86lb']
[2023-04-09T03:27:29.134+0000] {standard_task_runner.py:83} INFO - Job 99: Subtask store_user
[2023-04-09T03:27:29.195+0000] {task_command.py:376} INFO - Running <TaskInstance: api_dag.store_user manual__2023-04-09T03:27:22.565474+00:00 [running]> on host c12a518d534c
[2023-04-09T03:27:29.248+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2023-04-09T03:27:22.565474+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-09T03:27:22.565474+00:00
[2023-04-09T03:27:29.250+0000] {postgres.py:129} INFO - Running copy expert: COPY user_details FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2023-04-09T03:27:29.259+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-04-09T03:27:29.267+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-04-09T03:27:29.278+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=api_dag, task_id=store_user, execution_date=20230409T032722, start_date=20230409T032729, end_date=20230409T032729
[2023-04-09T03:27:29.303+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-09T03:27:29.319+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
